tsconfig.json

```diff
@@ -1,10 +1,12 @@
 {
   "compilerOptions": {
     "allowJs": false,
-    "target": "es2020",
+    "target": "es2022",
```

Comment:
Please revert these ES target changes. Bumping this would mean we will not be able to support some older versions of Nodejs. This should be treated separately in a backwards-compatible way once we move to Es Modules/